gradle.properties

```diff
@@ -1,4 +1,4 @@
 // Project properties
-version=2.0.0-BETA
+version=2.0.0
```

Comment:
There are missing STABLE release commits in this PR. Stable branch is at version 2.0.1, for example.

Reply:
@littleaj did you pulled the latest stable before merging locally?

Reply:
@dhaval24 nope :) fixed.